R/process_extract_alarms_telecare.R

```diff
@@ -4,33 +4,32 @@
 #' (year specific) Alarms Telecare extract, it will return the final data
 #' but also write this out as rds.
 #'
-#' @param data The extract to process. (Optional) Can be passed through a data list or
-#' alternatively read the file from disk.
+#' @param data The extract to process - Read the file from disk.
 #' @param year The year to process, in FY format.
-#' @param client_lookup The client lookup extract (Optional) Can be passed through a data list
-#' or alternatively read the file from disk.
+#' @param client_lookup The client lookup extract - Read the file from disk.
 #' @param write_to_disk (optional) Should the data be written to disk default is
 #' `TRUE` i.e. write the data to disk.
 #'
 #' @return the final data as a [tibble][tibble::tibble-package].
 #' @export
 #' @family process extracts
-process_extract_alarms_telecare <- function(data = NULL, year, client_lookup = NULL, write_to_disk = TRUE) {
-  # Include is.null for passing the processed ALL alarms Telecare data through a list
-  if (is.null(data)) {
-    data <- readr::read_rds(get_sc_at_episodes_path())
-  }
-
+process_extract_alarms_telecare <- function(data = get_sc_at_episodes_path(),
+                                            year,
+                                            client_lookup = get_source_extract_path(year, type = "Client"),
```

Comment:
I don't like having parameters called data which are expecting a file path. Can we rename them please?
```suggestion
process_extract_alarms_telecare <- function(file_path = get_sc_at_episodes_path(),
                                            year,
                                            client_lookup_path = get_source_extract_path(year, type = "Client"),
```